pkg/exchange/bybit/stream.go

```diff
@@ -27,35 +27,49 @@ var (
 	wsAuthRequest = 10 * time.Second
 )
 
+//go:generate mockgen -destination=mocks/stream.go -package=mocks . MarketInfoProvider
+type MarketInfoProvider interface {
+	GetAllFeeRates(ctx context.Context) (bybitapi.FeeRates, error)
+	QueryMarkets(ctx context.Context) (types.MarketMap, error)
+}
+
 //go:generate callbackgen -type Stream
 type Stream struct {
-	key, secret string
 	types.StandardStream
 
+	key, secret    string
+	marketProvider MarketInfoProvider
+	// TODO: update the fee rate at 7:00 am UTC; rotation required.
+	symbolFeeDetails map[string]*symbolFeeDetail
```

Comment:
Will this be accessed from multiple goroutines?

Reply:
Currently, data is only written during the initialization connection, so I haven't planned to handle it yet. When I get around to completing the TODOs, I will address it